src/proto/message_translation/tbots_geometry_test.cpp

```diff
@@ -69,13 +69,14 @@ TEST(TbotsProtobufTest, circle_msg_test)
 
 TEST(TbotsProtobufTest, velocity_obstacle_msg_test)
 {
-    VelocityObstacle velocity_obstacle;
-    velocity_obstacle.apex_      = Vector(3, 3);
-    velocity_obstacle.side1_     = Vector(-1, -2);
-    velocity_obstacle.side2_     = Vector(1, 2);
-    auto velocity_obstacle_msg_1 = createVelocityObstacleProto(velocity_obstacle);
-    auto velocity_obstacle_2     = createVelocityObstacle(*velocity_obstacle_msg_1);
-    auto velocity_obstacle_msg_2 = createVelocityObstacleProto(velocity_obstacle_2);
+    Vector apex_  = Vector(3, 3);
+    Vector side1_ = Vector(-1, -2);
+    Vector side2_ = Vector(1, 2);
+
+    VelocityObstacle velocity_obstacle = VelocityObstacle(apex_, side1_, side2_);
```

Comment:
nit: The underscore at the end of variables can be removed

Reply:
updated